src/docs/testing/ui-performance/index.md

```diff
@@ -412,49 +388,51 @@ At this point, disable the graphs and checkerboardOffScreenLayers.]
 
 Run the app and look for images rendered with a randomly colored checkerboard,
 indicating that the image is cached. As you interact with the scene,
-the checkerboarded images should remain constant—you don’t want to see
+the checkerboarded images should remain constant&mdash;you don’t want to see
 flickering, which would indicate that the cached image is being re-cached.
 
 In most cases, you want to see checkerboards on static images,
 but not on non-static images.  If a static image isn't cached,
-you can cache it by placing it into a
-[RepaintBoundary]({{site.api}}/flutter/widgets/RepaintBoundary-class.html)
+you can cache it by placing it into a [RepaintBoundary][]
 widget. Though the engine may still ignore a repaint boundary if it
 thinks the image isn't complex enough.
 
+### Viewing the widget rebuild profiler
+
+You can view the widget rebuilt counts for the current screen and
+frame in the widget inspector. For details on how to do this, see
+[Show performance data][] in the [Android Studio / IntelliJ][] page.
+
 ## Debug flags
 
 Flutter provides a wide variety of debug flags and functions to help you
 debug your app at various points along the development cycle.
 To use these features, you must compile in debug mode.
 The following list, while not complete,
 highlights some of the more useful flags (and one function)
-from the [rendering
-library]({{site.api}}/flutter/rendering/rendering-library.html)
-for debugging performance issues.
+from the [rendering library][] for debugging performance issues.
 
-* [`debugDumpRenderTree()`]({{site.api}}/flutter/rendering/debugDumpRenderTree.html)<br>
+* [`debugDumpRenderTree()`][]<br>
```

Comment:
Can we please not tell users to use debugDumpRenderTree? The text takes too long to dump and is barely readable. Users should be using the inspector instead in 99% of cases.